tests/visualize/create/test_patches.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from visualize.create.patches import (
+    ColorRgb,
+    _bbox_color,
+    RED_RGB,
+    GREEN_RGB,
+    YELLOW_RGB,
+)
+
+
+@pytest.mark.parametrize(
+    "similarity,expected_color",
+    [
+        (0.0, RED_RGB),
+        (0.2, (255, 102, 0)),
+        (0.5, YELLOW_RGB),
+        (0.6, (204, 255, 0)),
+        (0.9, (50, 255, 0)),
+        (1.0, GREEN_RGB),
+    ],
+)
+def test_bbox_color_happy(similarity: float, expected_color: ColorRgb):
```

Comment:
You can keep this test but tbh. I don't think tests of this type are too useful. This essentially tests some configuration, which in fact can't be tested. If we decide to switch some colors, nothing bad happens but the test fails. An integration test that builds documentation pages and displays the final result would do a better job, and no private methods need to be accessed there. Eventually, we will have such integration tests (probably in form of notebooks)

Reply:
Yeah I agree, these sorts of tests aren't that helpful; I'd written them partially to check my thinking when writing the code but I guess at this point they're just going to ossify the code. I've removed them now.

I've got this outstanding issue (https://github.com/appliedAI-Initiative/ProtoTree/issues/23) to add some proper tests/mock data that actually exercise the main code.